### ISSUE TITLE
Vendors catalog don't use Schema.org

### ISSUE THREAD

**Issue by laudrain** [benetech/UX-Guide-EPUB-A11y-Metadata#4]
in the section "Exposing EPUB Accessibility Metadata to Search Engines":
> If an ebook catalogue is publicly available, metadata related to the accessibility of the contents can be displayed to search engine crawlers. 

The **Implementation** sub-section expose Schema.org only, but vendors use ONIX feeds from publishers to ingest titles in their catalogs, not schema.org.

**Comment by gregoriopellegrino** [comment]
You're right: Google, although it suggests to implementers to use Schema.org to collect information about books (https://developers.google.com/search/docs/data-types/book) actually uses ONIX streams (I think the ones received through the Google Books program and not from web pages, right?).

That said: about 22% of web traffic is generated by (good) bots (https://www.imperva.com/blog/bot-traffic-report-2016/?utm_campaign=Incapsula-moved), so I think it might be useful to expose structured information on the accessibility features of documents. We don't know what information these bots collect, but it may be that someone collects structured data and maybe information about accessibility; if not now it will probably happen in the future.

My proposal is to change the sentence to:

> If an ebook catalogue is publicly available, metadata related to the accessibility of the contents can be displayed to crawlers or bots that can collect it in a structured way.

**Comment by clapierre** [comment]
I am fine with this clarification.

**Comment by gregoriopellegrino** [comment]
Sorry, comment removed.

**Comment by gregoriopellegrino** [comment]
Correct. The idea discussed in the Working Group is that frontend platforms digest EPUBs and extract Schema.org information directly from the file.

However, there are some case studies where this is not possible, for example for lending platforms that do not have the file, but only the ONIX metadata, and that require the file from the digital distributor ONLY when the user requests it.

**Comment by laudrain** [comment]
@gregoriopellegrino even for the first use case where front end platforms ingest EPUB files, do we know which ones don't receive any ONIX feeds ?

Also, as EPUB has nothing on commercial information, and minimal on bibliographic metadata, how do these build their catalog entries?

**Comment by gregoriopellegrino** [comment]
You're right, I think we need to have an open discussion at the next call about this document. If Charles agrees, I would invite you to attend.

**Comment by diagram-codesprint** [comment]
I totally agree lets bring this up in the next call.  I will probably not be on the call but whatever you all decide I am sure will be fine, and feel free to submit a pull request with the changes and I can review when I get back from Vacation on July 10th.

**Comment by clapierre** [comment]
I totally agree lets bring this up in the next call. We will discuss this one June 19'th call
